### ISSUE TITLE
train loss 很低

### ISSUE THREAD

**Issue by jianminglv20** [yijingru/BBAVectors-Oriented-Object-Detection#102]
作者，您好！我使用您的代码测试我自己的数据集，刚开始训练时train loss就很低，不知道是否是batchsize设置太小的原因？一块GPU不能训练此代码吗？还是需要改些参数！

(dota) xiaobu@xiaobu-MS-7C82:~/project/BBAVectors-Oriented-Object-Detection-master$ python main.py --data_dir ./planesplit1 --num_epoch 100 --batch_size 3 --dataset dota --phase train
Setting up data...
Starting training...
----------
Epoch: 1/100 
train loss: 1.0731722713907137
/home/xiaobu/anaconda3/envs/dota/lib/python3.6/site-packages/torch/optim/lr_scheduler.py:143: UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)
----------
Epoch: 2/100 
train loss: 0.0006129515672007095
----------
Epoch: 3/100 
train loss: 0.00011045779322242325

**Comment by yijingru** [comment]
Overfit?
